website/docs/whats-new/release-notes/partials/components.md

```diff
@@ -0,0 +1,200 @@
+ **`npm package: @hashicorp/design-system-components`**
+
+## 2.12.2 <Doc::Badge @type="outlined">Latest</Doc::Badge>
+
+**September 22nd, 2023**
+
+- `PowerSelect` fix style overrides when the list is positioned above. [(#1655)](https://github.com/hashicorp/design-system/pull/1655) Thanks [@alex-ju](https://github.com/alex-ju)! 
+
+- `Text` fixed issue with whitespace adding extra underline when used in links (eg. inside a `Link::Inline`)[(#1652)](https://github.com/hashicorp/design-system/pull/1652) Thanks [@didoo](https://github.com/didoo)! 
+
+- 🔄 Updated dependencies:
+    - @hashicorp/ember-flight-icons@3.1.3
+
+## 2.12.1
+
+**September 19th, 2023**
+
+- `Pagination` bugfix `aria-label` on the component. [(#1627)](https://github.com/hashicorp/design-system/pull/1627) Thanks [@MelSumner](https://github.com/MelSumner)! 
+
+- `Dropdown` changed `@height` property to use `max-height` instead of a fixed height.[(#1635)](https://github.com/hashicorp/design-system/pull/1635) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 
```

Comment:
```suggestion
- `Pagination` - fixed a bug with `aria-label` on the component. [#1627](https://github.com/hashicorp/design-system/pull/1627) Thanks [@MelSumner](https://github.com/MelSumner)! 

- `Dropdown` - changed the `@height` property to use `max-height` instead of a fixed height. [#1635](https://github.com/hashicorp/design-system/pull/1635) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 
```